cmp/grid/GridModel.ts

```diff
@@ -233,34 +240,34 @@ export interface GridConfig {
 
     /**
      * Callback when a row is clicked - will receive an event with a data node containing
-     * the row's data. (Note that this may be null - e.g. for clicks on full-width group rows.)
+     * the row's data. (Note that the event may be null - e.g. for clicks on full-width group rows.)
      */
-    onRowClicked?: (e: any) => void;
+    onRowClicked?: (e: RowClickedEvent) => void;
 
```

Comment:
All of the text in docstring describing what is passed to the callback is obsolete -- typescript type best defines it

Reply:
Removed all texts in comments pertaining to the now-typed parameters of the functions.